packages/web-workers/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
Is this a correct place? Should I create new block for current release version and date? cc @lemonmade

Reply:
Yes, please create a block for the new version and date